Services/DataX.Flow/Flow.ManagementService/FlowManagementServiceStartup.cs

```diff
@@ -0,0 +1,138 @@
+        }
+
+        // Get all the dependencies needed to fulfill the ConfigGen
+        // requirements for cloud mode
+        private IList<Assembly> CloudModeDependencyAssemblies
+            => new List<Assembly>()
+                {
+                    typeof(DataX.Config.ConfigGenConfiguration).Assembly,
+                    typeof(DataX.Config.ConfigurationProviders.CosmosDbConfigurationProvider).Assembly,
+                    typeof(DataX.Config.Storage.CosmosDBConfigStorage).Assembly,
+                    typeof(DataX.Config.KeyVault.KeyVaultClient).Assembly,
+                    typeof(DataX.Config.LivyClient.LivyClientFactory).Assembly,
+                    typeof(DataX.Config.Input.EventHub.Processor.CreateEventHubConsumerGroup).Assembly
+                };
+
+        // Get all the dependencies needed to fulfill the ConfigGen
+        // requirements for oneBox mode
+        private IList<Assembly> OneBoxModeDependencyAssemblies
+            => new List<Assembly>()
+                {
+                    typeof(DataX.Config.ConfigGenConfiguration).Assembly,
+                    typeof(DataX.Config.Local.LocalDesignTimeStorage).Assembly
+                };
+
+        // Get the required settings to bootstrap the config gen
+        private void InitConfigSettings()
+        {
+            InitialConfiguration.Set(DataX.Config.ConfigDataModel.Constants.ConfigSettingName_EnableOneBox, Settings.EnableOneBox.ToString());
+
+            if (!Settings.EnableOneBox)
+            {
+                InitialConfiguration.Set(CosmosDbConfigurationProvider.ConfigSettingName_CosmosDBConfig_ConnectionString, Settings.CosmosDBConfigConnectionString);
+                InitialConfiguration.Set(CosmosDbConfigurationProvider.ConfigSettingName_CosmosDBConfig_DatabaseName, Settings.CosmosDBConfigDatabaseName);
+                InitialConfiguration.Set(CosmosDbConfigurationProvider.ConfigSettingName_CosmosDBConfig_CollectionName, Settings.CosmosDBConfigCollectionName);
+                InitialConfiguration.Set(DataX.Config.ConfigDataModel.Constants.ConfigSettingName_ServiceKeyVaultName, Settings.ServiceKeyVaultName);
+            }
+            else
+            {
+                // Local settings
+                var metricsHttpEndpoint = Settings.MetricsHttpEndpoint.TrimEnd('/') + _MetricsHttpEndpointRelativeUri;
+                InitialConfiguration.Set(DataX.Config.ConfigDataModel.Constants.ConfigSettingName_LocalRoot, Settings.LocalRoot);
+                InitialConfiguration.Set(DataX.Config.Local.LocalSparkClient.ConfigSettingName_SparkHomeFolder, Settings.SparkHome);
+                InitialConfiguration.Set(DataX.Config.ConfigDataModel.Constants.ConfigSettingName_ClusterName, "localCluster");
+                InitialConfiguration.Set(DataX.Config.ConfigDataModel.Constants.ConfigSettingName_ServiceKeyVaultName, "local");
```

Comment:
Do these need to be fully qualified?  I'd rather avoid having DataX.Config everywhere if possible

Reply:
Nope. This was a direct copy-paste from the original code before changes occurred. I'll add a using to make it less verbose.